integration-test/src/main/resources/testsuites/smoke/yarn-smoke-tests.yaml

```diff
@@ -0,0 +1,15 @@
+name: "yarn-smoke-tests"
+parameters:
+  cloudProvider: YARN
```

Comment:
It's not cloudProvider, but integrationtest.cloudProvider. Even then, I'm not sure this will work. Has to be tried.

Reply:
Unfortunately it did not...

Reply:
We should ask @lnardai or @topolyai5 about this feature. They maybe know something about it.